lib/std/json/stringify.zig

```diff
@@ -510,6 +510,9 @@ pub fn WriteStream(
                     return self.write(@as(std.math.IntFittingRange(value, value), value));
                 },
                 .float, .comptime_float => {
+                    if (value == std.math.inf(f32)) {
+                        return error.InvalidJSON;
+                    }
```

Comment:
may also want to check `std.math.isNan` while we're here

Reply:
Great suggestion! I added a Nan check too!